pkg/controller/nodelifecycle/metrics.go

```diff
@@ -79,6 +79,25 @@ var (
 		},
 		[]string{"zone"},
 	)
+
+	updateNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_node_health_duration_seconds",
+			Help:           "Duration in seconds for NodeController to update the health of a single node.",
+			Buckets:        metrics.ExponentialBuckets(0.001, 2, 16), // 1ms -> ~30s
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+	monitorNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_nodes_health_duration_seconds",
```

Comment:
Same. Metrics names should be defined as const similar to other ones. i.e. evictionsTotalKey

Reply:
I'll defer to @kubernetes/sig-instrumentation-approvers for verdict, but I think the latter metrics should be sufficient on this.

Reply:
I would definitely recommend to at least changing the name of the metrics since they will be quite confusing and prone to errors to end-users.

@soltysh do you mean that this metric should be sufficient or the other one?

My personal opinion is that `update_node_health_duration_seconds` is the most useful one here since it gives us an idea of the usual time it takes for a node health to be updated. As for the metric measuring the time it takes to update the health of all nodes, I don't really like it since it is not very actionable and could easily be inferred from the node-level one.

Reply:
>  Whereas the metric measuring the time it takes to update the health of all nodes is not very actionable and could easily be inferred from the node-level one.

It is potentially actionable e.g. that we need more workers to parallelize more.
But I don't feel super strongly about this.

Reply:
I didn't think about that scenario, I guess that make sense. I would just recommend updating the names then so that metrics are easier to distinguish by the users.

Reply:
> It is potentially actionable e.g. that we need more workers to parallelize more.

This was my thinking, that it would be easiest to understand the overall performance of  `monitorNodeHealth` with a metric that included everything.

> I would just recommend updating the names then so that metrics are easier to distinguish by the users.

I struggled to name these initially, hopefully the updated names are more obvious to end users. Definitely open to suggestions.